selfdrive/car/honda/carstate.py

```diff
@@ -192,6 +190,13 @@ def get_can_signals(CP, gearbox_msg="GEARBOX"):
     signals.append(("INTERCEPTOR_GAS2", "GAS_SENSOR", 0))
     checks.append(("GAS_SENSOR", 50))
 
+  if CP.openpilotLongitudinalControl:
+    signals += [
+      ("BRAKE_ERROR_1", "STANDSTILL", 1),
+      ("BRAKE_ERROR_2", "STANDSTILL", 1)
+    ]
+    checks += [("STANDSTILL", 50)]
+
```

Comment:
I believe this can yield a redundant STANDSTILL check in some cases.

Fall-through in an `if` block above is:
```python
    signals += [("DOOR_OPEN_FL", "DOORS_STATUS", 1),
                ("DOOR_OPEN_FR", "DOORS_STATUS", 1),
                ("DOOR_OPEN_RL", "DOORS_STATUS", 1),
                ("DOOR_OPEN_RR", "DOORS_STATUS", 1),
                ("WHEELS_MOVING", "STANDSTILL", 1)]
    checks += [
      ("DOORS_STATUS", 3),
      ("STANDSTILL", 50),
    ]
```